community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/spi/PlanContext.scala

```diff
@@ -35,7 +35,7 @@ trait PlanContext extends TokenContext {
 
   def getUniqueIndexRule(labelName: String, propertyKey: String): Option[IndexDescriptor]
 
-  def getUniquenessConstraint(labelName: String, propertyKey: String): Option[UniquenessConstraint]
+  def getUniquenessConstraint(labelName: String, propertyKey: String): Option[PropertyConstraint]
```

Comment:
Method name contains 'Uniqueness' in it's name, maybe return type should remain `Option[UniquenessConstraint]` ?